TeamCode/src/main/java/org/firstinspires/ftc/teamcode/test.java

```diff
@@ -0,0 +1,94 @@
+package org.firstinspires.ftc.teamcode;
+
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.acmerobotics.dashboard.telemetry.TelemetryPacket;
+import com.qualcomm.hardware.ams.AMSColorSensor;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Gamepad;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.teamcode.hardware.OrbitGyro;
+import org.firstinspires.ftc.teamcode.robotData.GlobalData;
+import org.firstinspires.ftc.teamcode.robotSubSystems.RobotState;
+import org.firstinspires.ftc.teamcode.robotSubSystems.SubSystemManager;
+import org.firstinspires.ftc.teamcode.robotSubSystems.arm.Arm;
+import org.firstinspires.ftc.teamcode.robotSubSystems.claw.Claw;
+import org.firstinspires.ftc.teamcode.robotSubSystems.drivetrain.Drivetrain;
+import org.firstinspires.ftc.teamcode.robotSubSystems.elevator.Elevator;
+import org.firstinspires.ftc.teamcode.robotSubSystems.intake.Intake;
+
+@TeleOp(name = "test")
+public class test extends LinearOpMode {
+
+    private static ElapsedTime time = new ElapsedTime();
+
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+
+        FtcDashboard dashboard = FtcDashboard.getInstance();
+        TelemetryPacket packet = new TelemetryPacket();
+
+        time.reset();
+        Drivetrain.init(hardwareMap);
+        OrbitGyro.init(hardwareMap);
+        Elevator.init(hardwareMap);
+        Claw.init(hardwareMap);
+        Arm.init(hardwareMap);
+        Intake.init(hardwareMap);
+
+        GlobalData.inAutonomous = false;
+
+        waitForStart();
+
+        while (!isStopRequested()){
+
+        }
+    }
+
+
+    private static void testMotor(DcMotor motor, float power){
+        motor.setPower(power);
+    }
+
+    private static void testServo(Servo servo){
+        for (int i = 0; i < 3; i++){
+            time.reset();
+            servo.setPosition(45);
+            if (time.milliseconds() < 1000){}
+            time.reset();
+            servo.setPosition(0);
+            if (time.milliseconds() < 1000){}
+        }
+    }
+
```

Comment:
there is no need to for loop this test.

test servo should get initial pose and other pose.
for example:
```
  private static void testServo(Servo servo, float startPos, float otherPos){
            time.reset();
            servo.setPosition(otherPos);
            if (time.milliseconds() < 1000){}
            time.reset();
            servo.setPosition(startPos);
            if (time.milliseconds() < 1000){}
        }
```